src/abstractdataframe/selectionfast.jl

```diff
@@ -349,21 +391,17 @@ function table_transformation(df_sel::AbstractDataFrame, ::typeof(maximum))
     return reduce(max, map(identity, eachcol(df_sel)))
 end
 
-# TODO:
-# Add these transformations in the future
```

Comment:
@nalimilan: the question about removal of this was lost somewhere. I implemented the transformations that seemed useful. For instance I skipped `first` as it did not seem very common to use in case of row-wise aggregation (usually the user knows what the first selected column is). Also - since the PR is now fully general if someone really wants to do `first` one can add `collect` to composed function and have it. I did not want to add a lot of definitions that would not be used in practice.

Reply:
Yeah that was an outdated comment so I had removed it after posting it by mistake. :-)